src/app/core/store/effects/servables.effects.ts

```diff
@@ -35,7 +36,11 @@ export class ServablesEffects {
     exhaustMap(({ name }) =>
       this.servablesService.delete(name).pipe(
         map(() => deleteServableSuccess({ name })),
-        catchError(error => of(deleteServableFailed({ error })))
+        catchError(error => of(deleteServableFailed({ error })).pipe(
```

Comment:
You can simplify that block.
Just show snackbar before returning action's stream, we will remove superfluous 'pipe' and make it easier to read
```
catchError(err => {
  this.snackbarService.show(...)
  return of(...)
} ) 
```